docs/advanced-usage/satellite-domains.mdx

```diff
@@ -3,8 +3,10 @@ title: Authentication across different domains
 description: Learn how to share sessions across different domains by adding satellite domains to your application.
 ---
 
-> [!IMPORTANT]
-> This guide addresses authentication across different domains with shared sessions, for example `example-site.com` and `example-site-admin.com`. Authentication across subdomains with shared sessions works by default with Clerk.
+> [!WARNING]
```

Comment:
The other one uses "caution" and this one is "warning" - do we want to align them?